lib/msf/core/post/windows.rb

```diff
@@ -1,4 +1,50 @@
 # -*- coding: binary -*-
 
 module Msf::Post::Windows
+  include Msf::Post::Common
+
+  def initialize(info = {})
```

Comment:
This code may benefit from living in a specific class such as `Msf::Post::Windows::Drives` or similar, - as your suggested fix will still won't work for Python Meterpreter on windows

Reply:
The original internal jira ticket discussed https://github.com/rapid7/metasploit-framework/blob/7cbe70b05c3054c095575e078482ca2d33e59caf/lib/msf/core/post/windows/file_system.rb as an alternative place, but it suffers the same fate

Reply:
This code already falls back to `wmic` if railgun is not available, should `stdapi_railgun_api` env be listed here?